customizations.md

```diff
@@ -115,33 +115,43 @@ In the second option, a fleet manager would configure the map file on ECUs such
 
 ## Custom installation instructions for ECUs
 
-Most inputs to ECUs are delivered as signed Targets files and are stored on the Image directory. These signed Targets files are then sent to the ECU by the Director. However, there may be some cases where the inputs required for a particular customization cannot be configured to follow this standard signing process. Variations in input may be due to not knowing the input in advance, or a need to customize instructions for each vehicle. Examples of such inputs could be a command line option that turns on a feature in certain ECUs, a configuration sent by a Director repository to an ECU, or a Director doing a dynamic customization for an ECU. We can collectively call all these non-standard inputs "dynamic directions." Uptane allows ECUs to access dynamic directions in two different ways, each having particular advantages for different use cases.
+Most inputs to ECUs are delivered as signed Targets files, stored on the Image directory, and then sent to the ECU by the Director. However, there may be some cases where the inputs required for a particular customization cannot be configured to follow this standard signing process. Variations in input may be due to not knowing the input in advance, or a need to customize instructions for each vehicle. Examples of such inputs could be a command line option that turns on a feature in certain ECUs, a configuration sent by a Director repository to an ECU, or a Director doing a dynamic customization for an ECU. We can collectively call all these non-standard inputs "dynamic directions." Uptane allows ECUs to access dynamic directions in two different ways, each having particular advantages for different use cases.
 
 ### Accessing dynamic directions through signed images from the Director repository
 
-The first option for providing dynamic directions is to slightly modify the standard delivery procedure described above. A signed image would still be sent to the ECU from the Director repository, but with one main difference. Though this file will be signed by the Director, Timestamp, and Metadata roles, it would not be stored on—-or validated by—-the Image repository. As the Image repository is controlled by offline keys, it can not validate a file created dynamically by the Director.
+The first option for providing dynamic directions is to slightly modify the standard delivery procedure described above. An  image signed by the Director, Timestamp, and Metadata roles would still be sent to the ECU from the Director repository, but this file would not be stored on—-or validated by—-the Image repository. As the Image repository is controlled by offline keys, it can not validate a file created dynamically by the Director.
```

Comment:
Good question. Maybe we just pull out those references.
```suggestion
The first option for providing dynamic directions is to slightly modify the standard delivery procedure described above. The Director repository would still send a signed image to the ECU, but this file would not be stored on—-or validated by—-the Image repository. As the Image repository is controlled by offline keys, it can not validate a file created dynamically by the Director.
```

Reply:
Your suggestion looks good to me, thanks!